src/borg/helpers/parseformat.py

```diff
@@ -472,6 +472,27 @@ class Location:
         re.VERBOSE,
     )
 
+    # (s3|b2):[profile|(access_key_id:access_key_secret)@][schema://hostname[:port]]/bucket/path
+    s3_re = re.compile(
+        r"""
+        (?P<s3type>(s3|b2)):
```

Comment:
hmm, where is s3type used?

Reply:
Added back to be used in the proto field.